store/src/main/java/com/issoft/store/commands/OrderCommand.java

```diff
@@ -0,0 +1,41 @@
+package com.issoft.store.commands;
+
+import com.issoft.store.Product;
+import com.issoft.store.RandomStorePopulator;
+import com.issoft.store.threads.CleanUpCollectionThread;
+import com.issoft.store.threads.PurchaseGoodsThread;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+
+public class OrderCommand implements Command {
+
+    private static List<Product> selectedProducts;
+
+    @Override
+    public void execute(List<Product> products) throws IOException {
+        System.out.println("What Product do you want to buy? Please enter the number of the Product from the Product List. All available Products you can observe using Sort and Top commands.");
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        String userChoice = reader.readLine();
+        int productIndex = Integer.parseInt(userChoice) - 1;
+        List<Product> selectedProductsByUser = products.subList(productIndex, productIndex+1);
```

Comment:
what about using products.get() method instead of products.subList() ?
Do I understand correctly that the goal is to get element from collection by index ?